gnosis/gp-swap-ui
# Price quote/refactor state

Second of several PRs for including our own price estimation in CowSwap

Depends on https://github.com/gnosis/gp-swap-ui/pull/600

## Scope
This PR also don't change any behaviour in CowSwap. 

The goal is to refactor the fee state into another more general state that holds all quote information.
The new state is in `state/price`. 

The quote state after in this PR only has the fee information (and last check), but a future PR will add there the price for the same token pair and amount as the fee.

## Additional changes
This PR changes the polling checks to every 10 seconds, also sets some TODOs in preparation for next PR that will fetch the price.

## Test
For now, let's just test that this PR doesn't break the fee estimation.

1. Select token pair
2. Input amount
3. Check the network, verify the request for the fee was done, and that is shown in the UI
